content/F2015/F15-event-teaparty.md

```diff
@@ -0,0 +1,21 @@
+title: Big CSters Board Game and Tea Party Social
+Date: 2015-09-24
+Category: Events
+Tags: social, big csters
+Slug:  tea-party-F15
+Author: Ziwei Zhao
+Summary:  We'll be meeting up in the CS lounge for a tea party social!
+
+Come out for an evening of tea, snacks and board games with fellow women in CS!
+
+Please bring your own mug if you can!
+
+**About Big CSters:** Our Big CSters program helps connect cis & trans women and non-binary students in computer science and related fields, through events geared towards mentorship and community-building.
```

Comment:
This line is 200+ characters long. It's better to keep them to 80 characters or less because git calculates change sets on a line by line basis, so if the lines are very very long, it makes it harder to pinpoint where the change is and adds noise to the logs.

This is a small thing anyways, doesn't really matter.